ROADMAP.md

```diff
@@ -74,12 +74,13 @@ Using `gfx-rs` for backend abstraction allows us to provide compatibility with m
 **Next steps**
  * Raw input support (winit?)
  * Gamepad support (winit?)
- * Input Mapping
+ * Input Mapping: Map input events (e.g key press) to abstract action events (`Jump`, `Forward`, ..)
```

Comment:
I guess we can use types and have text based configs. Maybe by using `serde`. Then for example we could serialize/deserialize a `HashMap<event::Event, MovementEvent>` to/from YAML (or something else `serde` supports).